packages/atlas/src/views/studio/YppDashboard/YppDashboard.styles.ts

```diff
@@ -81,3 +82,41 @@ export const StyledSvgAlertsInformative24 = styled(SvgAlertsInformative24)`
     fill: ${cVar('colorTextStrong')};
   }
 `
+export const StyledTab = styled(Tabs)`
```

Comment:
You missed divider below the tabs:
Yours:
![image](https://user-images.githubusercontent.com/51168865/209781889-14b3866c-98e3-4c15-aece-9d37fc145122.png)
Design: 
![image](https://user-images.githubusercontent.com/51168865/209781965-dd3560d3-3de9-4048-b0c7-5f39b9299a9f.png)